docs/proposals/node-agent/workload.md

```diff
@@ -0,0 +1,23 @@
+# Workloads
+
+The node agent amanges workload management. A workload is the tiniest unit in orka. It can be a container, a virtual machine, etc.
```

Comment:
```suggestion
The node agent manages workload management. A workload is the tiniest unit in orka. It can be a container, a virtual machine, etc.
```

Reply:
Fixed in Fixed in https://github.com/dev-sys-do/orka/pull/1/commits/162a3a8f1cd6aba48c05413934d6f8ad29df369f